src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/TestHelper.java

```diff
@@ -75,11 +75,15 @@ public void uploadZipFile(
         String metadataFile,
         final String destZipFilename,
         String testPrivateKeyDer
-    ) throws Exception {
-        byte[] zipFile =
-            createSignedZipArchiveWithRandomName(files, metadataFile, destZipFilename, testPrivateKeyDer);
-        CloudBlockBlob blockBlobReference = container.getBlockBlobReference(destZipFilename);
-        blockBlobReference.uploadFromByteArray(zipFile, 0, zipFile.length);
+    ) {
+        try {
+            byte[] zipFile =
+                createSignedZipArchiveWithRandomName(files, metadataFile, destZipFilename, testPrivateKeyDer);
+            CloudBlockBlob blockBlobReference = container.getBlockBlobReference(destZipFilename);
+            blockBlobReference.uploadFromByteArray(zipFile, 0, zipFile.length);
+        } catch (Exception exc) {
+            throw new RuntimeException(exc);
+        }
```

Comment:
so that I can use it in streams

Reply:
given the comment above, we won't need this change